LibVLCSharp.WPF/LibVLCSharp.WPF.csproj

```diff
@@ -50,6 +51,6 @@ LibVLC needs to be installed separately, see VideoLAN.LibVLC.* packages.
     </Page>
     <Page Remove="**\App.xaml" />
     <ApplicationDefinition Include="**\App.xaml" />
-    <Reference Include="System.Xaml" />
+    <Reference Include="System.Xaml" Condition="'$(TargetFramework)'=='net461'"/>
```

Comment:
I think this can be removed as well. Need to try a WPF netcore 3 sample to make sure